docs/guide/widgets.md

```diff
@@ -190,6 +190,67 @@ If the supplied text is too long to fit within the widget, it will be cropped (a
 There are a number of styles that influence how titles are displayed (color and alignment).
 See the [style reference](../styles/index.md) for details.
 
+## Focus & keybindings
+
+Widgets can have a list of associated key [bindings](../guide/input.md#bindings),
+which let them call [actions](../guide/actions.md) in response to key presses.
+
+A widget is only able to handle key presses if it or one of its descendants has [focus](../guide/input.md#input-focus).
+
+To demonstrate, let's design a simple interactive counter widget which can be incremented and decremented using the keyboard.
+
+In the following example, we define a simple `Counter` widget with `can_focus=True`, and some CSS to make it stand out when focused.
```

Comment:
the `can_focus=True` is a really important takeaway here. It's what we were lacking, and why we keep getting asked. Here it seems too much of a sidenote.

"Widgets aren't focusable by default. In order for a widget to be focusable we need to set `can_focus=True` when we define a widget subclass. Here's an example of a focusable widget ..."

Reply:
I think this is clearer in the rendered docs, as the example app below has a single highlighted line for the `can_focus=True` as well as a popup explaining it. The whole purpose of the example is to draw attention to the `can_focus=True`.

Have reworded to try and emphasise it more.